packages/venia-ui/lib/components/CartPage/ProductListing/product.js

```diff
@@ -84,37 +84,41 @@ const Product = props => {
 
     return (
         <li className={classes.root}>
-            <Link to={itemLink} className={classes.item}>
+            <Link to={itemLink} className={classes.imageContainer}>
                 <Image
                     alt={name}
                     classes={{
-                        image: classes.image,
-                        root: classes.imageContainer
+                        root: classes.imageRoot,
+                        image: classes.image
                     }}
                     width={IMAGE_SIZE}
                     resource={image}
                 />
-                <div className={classes.details}>
-                    <span className={classes.name}>{name}</span>
+            </Link>
+            <div className={classes.details}>
+                <Link to={itemLink} className={classes.name}>
+                    {name}
+                </Link>
+                <Link to={itemLink} className={classes.optionsContainer}>
                     <ProductOptions
                         options={options}
                         classes={{
                             options: classes.options,
                             optionLabel: classes.optionLabel
                         }}
                     />
-                    <span className={classes.price}>
-                        <Price currencyCode={currency} value={unitPrice} />
-                        {' ea.'}
-                    </span>
+                </Link>
+                <Link to={itemLink} className={classes.price}>
```

Comment:
Just to put it out there, I would only expect the image and the product name to be links to the PDP but not the options or price.

Reply:
I am awaiting UX's response on #2549 which is similar to this. Will update the comment here accordingly once I have an update.